llama_stack/apis/eval/eval.py

```diff
@@ -35,29 +36,49 @@ class AgentCandidate(BaseModel):
 ]
 
 
+@json_schema_type
+class BenchmarkEvalTaskConfig(BaseModel):
+    eval_candidate: EvalCandidate  # type: ignore
```

Comment:
why are there `# type: ignore` annotations?

Reply:
It was for some lint warnings not allowing variable annotations in type hints. 

<img width="437" alt="image" src="https://github.com/user-attachments/assets/435f5118-492e-4ff3-817e-adb4c4fe2627">

Reply:
yeah this is a pylance / pyright bug and happens because of `pytest.asyncio` apparently. we should not add any `# type: ignore` and just suffer for now